src/Routes/SandboxPage/SandboxPage.scss

```diff
@@ -0,0 +1,28 @@
+.sbx-c-card-banner {
+  background-image: url(https://console.redhat.com/apps/frontend-assets/background-images/developer-sandbox-ui/Containers-pattern-Blog-hero.png);
+  background-color: var(--pf-global--BackgroundColor--dark-100);
+  background-size: cover;
+  .pf-c-button {
+    text-align: left;
+    white-space: normal;
+  }
+}
+
+.pf-c-page__main-section {
+  max-width: 1250px;
```

Comment:
This feels kinda strange, what if the width of the screen is larger?

Reply:
Okay, I was a bit curious why it would be used. But good point about stretching them beyoud that point.